docs/CHANGELOG.md

```diff
@@ -20,6 +19,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Cannot modify CVE of existing flaws (OSIDB-3102)
 - Jira metadata collector is not deleting metadata on failure (OSIDB-3219)
+
+## [4.1.7] - 2024-08-15
```

Comment:
outdated date

Reply:
It is rather outdated PR. It should have already be merged but it was not. The release happened on that date